React/Modules/RCTViewPropertyMapper.m

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTViewPropertyMapper.h"
+#import "RCTAnimation.h"
+#import "RCTNativeAnimatedModule.h"
+#import "RCTNativeAnimationManager.h"
+#import <UIKit/UIKit.h>
+#import "RCTBridge.h"
+#import "RCTUIManager.h"
+
+@implementation RCTViewPropertyMapper {
+  CGFloat tX_;
+  CGFloat tY_;
+  CGFloat sX_;
+  CGFloat sY_;
+  CGFloat r_;
+}
+
+- (instancetype)initWithViewTag:(NSNumber *)viewTag {
+  self = [super init];
+  if (self) {
+    _viewTag = viewTag;
+    tX_ = 0;
```

Comment:
Can you document what all these properties are? Or maybe just give them longer/clearer names?